docs/tutorials/mastering.md

```diff
@@ -30,31 +13,621 @@ For the purpose of this guide, the following values will be used.
+                                          10.201.0.100
+gnbsim                  Running           10.231.204.24    Ubuntu 22.04 LTS
+                                          10.201.0.103
+                                          10.204.0.100
+                                          10.1.143.192
+ran-access-router       Running           10.231.204.220   Ubuntu 22.04 LTS
+                                          10.201.0.110
+                                          10.204.0.1
+                                          10.202.0.1
+user-plane              Running           10.231.204.121   Ubuntu 22.04 LTS
+                                          10.201.0.102
+                                          10.203.0.100
+                                          10.202.0.100
+                                          10.1.112.128
+```
+
+### IP Addresses
 The following IP addresses are used in this tutorial and must be present in the DNS Server that all hosts are using.
 
-| Name | IP Address | Purpose |
-| ---- | ---------- | ------- |
-| `juju-controller.mgmt` | 10.201.0.10 | Management address for Juju machine |
-| `control-plane.mgmt` | 10.201.0.11 | Management address for control plane cluster machine |
-| `user-plane.mgmt` | 10.201.0.12 | Management address for user plane cluster machine |
-| `gnbsim.mgmt` | 10.201.0.13 | Management address for the gNB Simulator cluster machine |
-| `api.juju-controller.mgmt` | 10.201.0.50 | Juju controller address |
-| `cos.mgmt` | 10.201.0.51 | Canonical Observability Stack address |
-| `amf.mgmt` | 10.201.0.100 | Externally reachable control plane endpoint for the AMF |
-| `control-plane-nms.control-plane.mgmt` | 10.201.0.101 | Externally reachable control plane endpoint for the NMS |
+| Name | IP Address   | Purpose |
+| ---- |--------------| ------- |
+| `juju-controller.mgmt` | 10.201.0.104 | Management address for Juju machine |
+| `juju-controller.mgmt` | 10.201.0.104 | Management address for Juju machine |
+| `control-plane.mgmt` | 10.201.0.101 | Management address for control plane cluster machine |
+| `user-plane.mgmt` | 10.201.0.102 | Management address for user plane cluster machine |
+| `gnbsim.mgmt` | 10.201.0.103 | Management address for the gNB Simulator cluster machine |
+| `api.juju-controller.mgmt` | 10.201.0.50  | Juju controller address |
+| `cos.mgmt` | 10.201.0.51  | Canonical Observability Stack address |
+| `amf.mgmt` | 10.201.0.52  | Externally reachable control plane endpoint for the AMF |
+| `control-plane-nms.control-plane.mgmt` | 10.201.0.53  | Externally reachable control plane endpoint for the NMS |
 | `upf.mgmt` | 10.201.0.200 | Externally reachable control plane endpoint for the UPF |
 
-## 1. Bootstrap all Machines
 
-This section covers how to install Juju and MicroK8s to act as the infrastructure for SD-Core.
+### Install DNS Server
+
+First, replace the content of /etc/netplan/50-cloud-init.yaml as following to configure mgmt interface ip address as `10.201.0.100`.
```

Comment:
Why don't we use DHCP for this?  Is there no way to assign IP addresses in LXD?

Reply:
At the moment, auto ip assignment does not work for me for bridge networks. I asked through the link:
https://chat.canonical.com/canonical/pl/i4yoqebd57yo3gtr1w6whxtzqy

Reply:
DHCP client runs on the first NIC only, the netplan needs to be configured in the Multipass VMs to run the DHCP clients for the other NICs.  I think, manual ip configuration seems more appropriate here as we need to know this specific ip to configure the nameserver in other VMs.

Reply:
I see.  This is where cloud-init and friends come in.  All that is easily doable via openstack, so I had hoped there would be a way to do it with multipass